src/index.html

```diff
@@ -2,53 +2,38 @@
 <html lang="en">
 
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Delicious Meals</title>
   </head>
 
   <body>
     <header>
       <nav>
-        <ul>
-          <li><span>Logo</span></li>
-          <li><a href="#">Desert</a></li>
-          <li><a href="#">Vegan</a></li>
-          <li><a href="#">Pasta</a></li>
-        </ul>
+        <span id="logo">Logo</span>
+        <ul id="menu"></ul>
       </nav>
     </header>
 
     <main>
-      <section>
-        <ul>
-          <li>
-            <img alt="image" />
-            <h2>title</h2>
-            <span>5 likes</span>
-            <button type="button">likes</button>
-            <button type="button">comments</button>
-            <button type="button">reservations</button>
-          </li>
-        </ul>
+      <section id="list">
+        <ul id="list-block"></ul>
       </section>
-      <section>
-        <div>
-          <button type="button">close</button>
-          <img alt="image" />
-          <h2>title</h2>
-          <p>description</p>
-          <h3>comments / reservations</h3>
-          <ul>
-            <li>03/11/2022 Alex: Really like</li>
-          </ul>
-          <h3>Add comment / Add reservation</h3>
-          <form>
+      <section id="popup">
+        <div id="popup-block">
+          <button id="popup-close" type="button">X</button>
+          <img id="popup-image" alt="image" />
+          <h2 id="popup-title"></h2>
```

Comment:
For a good SEO (Search Engine Optimization) best practices, I recommend you start by `h1` then `h2, h3`, etc

Reply:
Good insight indeed & I totally agree on that 👏🏻  However, what do you think if we make an `h1` for the app name just below the header? or perhaps beside the logo?

Reply:
Great idea, thanks